core/lib/makeswift/components/site-theme/base-colors.tsx

```diff
@@ -0,0 +1,34 @@
+import { themeToCssVars } from './to-css';
+
+export const colors = {
```

Comment:
TODO: employ https://github.com/mertasan/tailwindcss-variables to move this to the Tailwind config.